Comanche055/ASSEMBLY_AND_OPERATION_INFORMATION.agc

```diff
@@ -865,15 +860,15 @@
 # -VV-
 # POSITION8		XXXX.X NAUT MI		DP	BIT 1 OF LOW REGISTER =
 #									 -28
-#							4 x 6,373,338 x 2
+#							4 X 6,373,338 X 2
 #							--------------------
-#								1852
+#							      1852
 #							NAUT MI.
 #
 # -XX-
-# POSITION 9		XXX.XX NAUT MI		DP	BIT 1 OF LOW REGISTER =
```

Comment:
The bracketed information is in the [source](http://www.ibiblio.org/apollo/ScansForConversion/Colossus249/0021.pdf) for the Apollo 9 CM AGC.